os/net/mac/mac-sequence.h

```diff
@@ -45,6 +45,16 @@
 #ifndef MAC_SEQUENCE_H
 #define MAC_SEQUENCE_H
 
+/**
+ * brief       Initializes.
```

Comment:
* `\brief`
* How about "Initializes the MAC sequence number to a random value"?  Making it clear that calling it twice actually influences the value.